### DISCUSSION TITLE
Software Developer

### DISCUSSION THREAD

**Discussion by naderik** [WeAreMahsaAmini/Core#211] [Introductions]
I'm Kasra, I'm a software developer, I can help with React, Nodejs, and pretty much anywhere else where development work is needed.

**Comment by ghost** [comment]
https://github.com/WeAreMahsaAmini/Core/discussions/256

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163
